packages/postcss-reduce-idents/test/index.js

```diff
@@ -249,40 +249,48 @@ test(
 );
 
 test(
-  'should rename grid-template-areas and grid-area',
+  'should rename grid-template-areas, grid-area, grid-column and grid-row',
   processCSS(
     [
       'body{grid-template-areas:"head head" \n"nav  main"\n"nav  foot";}',
       'header { grid-area: head }',
       'nav{grid-area:nav}',
       'main{grid-area:main}',
+      'aside{grid-column:nav}',
+      'section{grid-row:main}',
       'footer{grid-area:foot}',
     ].join(''),
     [
       'body{grid-template-areas:"a a" "b c" "b d";}',
       'header { grid-area: a }',
       'nav{grid-area:b}',
       'main{grid-area:c}',
+      'aside{grid-column:b}',
+      'section{grid-row:c}',
       'footer{grid-area:d}',
     ].join('')
   )
 );
 
 test(
-  'should rename grid-template-areas and grid-area (uppercase)',
+  'should rename grid-template-areas, grid-area, grid-column and grid-row (uppercase)',
   processCSS(
     [
       'body{GRID-TEMPLATE-AREAS:"head head" \n"nav  main"\n"nav  foot";}',
       'header { GRID-AREA: head }',
       'nav{GRID-AREA:nav}',
       'main{GRID-AREA:main}',
```

Comment:
Could be worthwhile to add more tests for more properties like `grid-row-end` so nobbody forgets them in the future.